test/integration/enumerating-databases/list_databases.test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
```

Comment:
My understanding of the `integration` directory is that the directory structure should roughly match the `spec` repo.  I don't think there's a spec for `enumerating-databases`.  I'm not sure exactly where these tests belong, maybe @nbbeeken or @dariakp  would have thoughts.  

I was thinking perhaps in `auth` because they're specifically testing auth functionality right now.

Reply:
There is actually a top level enumerate-databases spec, which probably matches this the closest. I would name the file enumerate-databases.test.ts and put it in top level integration directory

Reply:
Ah, my bad.  I missed it because I only looked at the specs that were in sub-directories